src/getDataTransferItems.js

```diff
@@ -1,4 +1,5 @@
-export default function getDataTransferFiles(event, isMultipleAllowed = true) {
+/* @flow */
+export default function getDataTransferFiles(event: Object, isMultipleAllowed: boolean = true) {
```

Comment:
Let's add return types everythere for such cases: `function(arg: type): type {}`